DEVELOPER_GUIDE.md

```diff
@@ -556,6 +556,40 @@ Then, you need to apply patterns for git-secrets, you can install the AWS standa
 git secrets --register-aws
 ```
 
+#### Remote Backed Storage
```

Comment:
The developer guide has a TOC on the top that needs to be re-generated/updated.

![Screenshot 2024-10-16 at 6 40 13 AM](https://github.com/user-attachments/assets/8b712d0e-3321-43b6-b04d-87a37aa9c7c0)